pallets/weights/src/pallet_checkpoint.rs

```diff
@@ -86,10 +87,8 @@ impl pallet_asset::checkpoint::WeightInfo for WeightInfo {
     // Storage: Checkpoint TotalSupply (r:0 w:1)
     // Storage: Checkpoint Timestamps (r:0 w:1)
     // Storage: Checkpoint ScheduleRefCount (r:0 w:1)
-    fn create_schedule(s: u32) -> Weight {
-        (137_723_000 as Weight)
-            // Standard Error: 106_000
-            .saturating_add((518_000 as Weight).saturating_mul(s as Weight))
+    fn create_schedule(_s: u32, ) -> Weight {
+        (147_224_000 as Weight)
```

Comment:
A fixed value of `1` is passed to this weight function.  So ignoring it here and the new weights is fine.  But we should review the benchmarks for this.